x/chainlink/client/rest/txFeed.go

```diff
@@ -16,26 +17,31 @@ type createFeedRequest struct {
 	FeedData  string       `json:"feedData"`
 }
 
-func createFeedHandler(clientCtx client.Context) http.HandlerFunc {
+func submitFeedDataHandler(clientCtx client.Context) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		var req createFeedRequest
 		if !rest.ReadRESTReq(w, r, clientCtx.LegacyAmino, &req) {
 			rest.WriteErrorResponse(w, http.StatusBadRequest, "failed to parse request")
 			return
 		}
-
 		baseReq := req.BaseReq.Sanitize()
 		if !baseReq.ValidateBasic(w) {
 			return
 		}
 
-		submitter, err := sdk.AccAddressFromBech32(req.Submitter)
+		submitter, err := sdk.AccAddressFromBech32(baseReq.From)
```

Comment:
some of these params will need to change with the fixes to the feedData Structure that will be coming in from a different PR.

Reply:
but again, not to be done right now but eventually! ^^